frontend/src/static/js/v_summary.js

```diff
@@ -45,7 +45,7 @@ window.vSummary = {
       isSortingWithinDsc: '',
       filterTimeFrame: 'commit',
       filterBreakdown: false,
-      isMergeGroup: false,
+      mergedGroups: 'none',
```

Comment:
It seems the mergedGroups type is switch btw string and list of repos. Maybe it is better to standardise it. e.g., use null instead of string 'none'.

Reply:
Then maybe better to make it a list of int? And also can change the name to mergedGroupIndexes

Reply:
Btw is it possible to have a attribute like isMerged for each repo instead of record down the indexes?

Reply:
> Btw is it possible to have a attribute like isMerged for each repo instead of record down the indexes?

That is feasible but doing so will require change to the current representation of repos. Currently they are just arrays without any other attributes. Changing that may require large refactoring while benefits of it may not be so much.

Reply:
> Then maybe better to make it a list of int? And also can change the name to mergedGroupIndexes

When I first thought about the type of this variable I used `string` because it can easily bind to the `merge all groups` checkbox with `v-model`, i.e. `all` represents checked while anything else represents unchecked. Let's see whether others have any opinions on this?

Reply:
Since changing any of the sorting options or checking any file types will reset all the merge groups, perhaps we can store the merged references as the repoName (when `group by repos`) or authorName (when `group by authors`) instead of indexes? 

In this way, the sorting options or checking any file types will not affect the merged groups.

Reply:
> Since changing any of the sorting options or checking any file types will reset all the merge groups, perhaps we can store the merged references as the repoName (when group by repos) or authorName (when group by authors) instead of indexes?

Thanks for the suggestion. I have changed the implementation for `mergedGroups`.